ingest/README.md

```diff
@@ -0,0 +1,89 @@
+# nextstrain.org/ebola/ingest
+
+This is the ingest pipeline for Ebola virus sequences.
+
+## Usage
+
+> NOTE: All command examples assume you are within the `ingest` directory.
+> If running commands from the outer `ebola` directory, please replace the `.` with `ingest`
+
+Fetch sequences with
+
+```sh
+nextstrain build --cpus 1 . data/sequences_all.ndjson
+```
+
+Run the complete ingest pipeline with
+
+```sh
+nextstrain build --cpus 1 .
+```
+
+This will produce two files (within the `ingest` directory):
+
+- data/metadata_all.tsv
+- data/sequences_all.fasta
+
+Run the complete ingest pipeline and upload results to AWS S3 with
+
+```sh
+nextstrain build . --configfiles config/config.yaml config/optional.yaml
+```
+
+### Adding new sequences not from GenBank
+
+#### Static Files
+
+Do the following to include sequences from static FASTA files.
+
+1. Convert the FASTA files to NDJSON files with:
+
+    ```sh
+    wget https://raw.githubusercontent.com/nextstrain/monkeypox/master/ingest/bin/fasta-to-ndjson -O ingest/bin/fasta-to-ndjson
```

Comment:
> I was surprised to find that this section of the README assumes that the user is operating from the top-level with paths like `ingest/bin/fasta-to-ndjson`

Done with commit: d15e63c8e211606b6f57e8a09f3de96e362e7ecb

Reply:
> We might also prefer `curl` over `wget` for these kinds of downloads, since `curl` is more likely to be installed by default on most systems. [@tsibley](https://github.com/tsibley) might know for sure about this though...

Sure, I can change it. (Wish there was a way to propagate such changes across multiple Snakefiles ;) ) Expect multiple PRs across repos soonish.

But also some FYI:

* https://www.reddit.com/r/linux/comments/wll8yq/comment/iju76yn/?utm_source=share&utm_medium=web2x&context=3
* https://daniel.haxx.se/docs/curl-vs-wget.html

And after getting classes of 30 students to install bioinformatic software on Mac, Windows, and whatever flavor of linux from 2015 to 2017; sometimes `wget` was missing, sometimes `curl` was missing. Ditto on various HPCs. It's possible there's a clear winner in 2022? It's hard to stay up-to-date.

Reply:
Done with bb9421bacbb2a3ce5db48c22cd716041858c913f